kollect-extensions/src/main/kotlin/kollect/extensions/Kollect.kt

```diff
@@ -0,0 +1,33 @@
+package kollect.extensions
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.core.Tuple2
+import arrow.extension
+import arrow.typeclasses.Monad
+import kollect.Kollect
+import kollect.KollectPartialOf
+import kollect.KollectResult
+import kollect.fix
+
+// Kollect ops
+@extension
+interface KollectMonad<F, I> : Monad<KollectPartialOf<F>> {
+
+    fun MF(): Monad<F>
+
+    override fun <A> just(a: A): Kollect<F, A> = Kollect.Unkollect(MF().just(KollectResult.Done(a)))
+
+    override fun <A, B> Kind<KollectPartialOf<F>, A>.map(f: (A) -> B): Kollect<F, B> =
```

Comment:
We should avoid using `Kind` when we can provide a more specific (??) type :D 

```suggestion
    override fun <A, B> KollectOf<F, A>.map(f: (A) -> B): Kollect<F, B> =
```

Reply:
Done 👍